src/main.ts

```diff
@@ -27,6 +31,24 @@ async function capture(cmd: string, options?: CaptureOptions): Promise<string> {
   return output.trim()
 }
 
+let _lsb_release: Promise<string>
+async function lsb_release(): Promise<string> {
+  if (!_lsb_release) {
+    _lsb_release = capture('lsb_release -c -s', {silent: true})
+  }
+
+  return _lsb_release
+}
+
+let _httpc: httpm.HttpClient
+async function http_get(url: string): Promise<httpm.HttpClientResponse> {
+  if (!_httpc) {
+    _httpc = new httpm.HttpClient('httpc')
+  }
+  core.info('curl ' + url)
```

Comment:
Nit: We don't use `curl` here, so the log entry is imprecise. I would write it as `core.info('HTTP GET ' + url)`.

Reply:
Accepted